tests/unittests/nominal/test_cramers.py

```diff
@@ -0,0 +1,169 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import itertools
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+
+from torchmetrics.functional.nominal.cramers import cramers_v, cramers_v_matrix
+from torchmetrics.nominal.cramers import CramersV
+from torchmetrics.utilities.imports import _DYTHON_AVAILABLE
+from unittests.helpers.testers import BATCH_SIZE, NUM_BATCHES, MetricTester
+
+if _DYTHON_AVAILABLE:
+    from dython.nominal import cramers_v as dython_cramers_v
```

Comment:
```suggestion
from dython.nominal import cramers_v as dython_cramers_v
```
This shall be here always for testing so if it was failing, check if all testing dependencies are installed :rabbit: